python/lsst/meas/astrom/astrometry.py

```diff
@@ -75,7 +75,7 @@ class AstrometryConfig(RefMatchConfig):
         dtype=bool,
         doc=("If True then a rough zeropoint will be computed from matched sources "
              "and outliers will be rejected in the iterations."),
-        default=False,
+        default=True,
```

Comment:
I see that this now matches the LSST value and that the corresponding config setting has been removed in `obs_lsst`. However, do you want to add `doMagnitudeOutlierRejection=False` to `obs_subaru`, or are you purposely changing the behavior there?

Reply:
I have overridden it in obs_subaru, as part of a commit that tries to ensure HSC processing is unchanged after this ticket. It also has a note that they should review those new overrides, as they are likely unnecessary.